Doc/library/enum.rst

```diff
@@ -383,8 +383,8 @@ enumeration, with the exception of special methods (:meth:`__str__`,
 variable names listed in :attr:`_ignore_`.
 
 Note:  if your enumeration defines :meth:`__new__` and/or :meth:`__init__` then
-whatever value(s) were given to the enum member will be passed into those
-methods.  See `Planet`_ for an example.
+whatever value(s) given to the enum member will be passed into those methods.
```

Comment:
"were" is necessary in the original sentence.  However, if you also change "whatever" to "any" (while still removing "were") I think we'll have a better sentence.

Reply:
Agreed, will do.